pxtlib/package.ts

```diff
@@ -318,6 +318,24 @@ namespace pxt {
             return regex.test(ts);
         }
 
+        private upgradePackagesAsync() {
+            return pxt.packagesConfigAsync()
+                .then(config => {
+                    let numfixes = 0
+                    U.iterMap(this.config.dependencies, (pkg, ver) => {
+                        if (pxt.github.isGithubId(ver)) {
+                            const upgraded = pxt.github.upgradedPackageReference(config, ver)
```

Comment:
probably good to first normalize the GitHub id

Reply:
```
                            ver = pxt.github.normalizeRepoId(ver);
```

Reply:
we parse it on the other side, so it's probably OK